.changeset/empty-cherries-admire.md

```diff
@@ -0,0 +1,14 @@
+---
+'skuba': major
+---
+
```

Comment:
@72636c would you mind wordsmithing this? I'll proceed to merge without significant manual testing in SEEK repos; I hope to be able to get out https://github.com/seek-oss/skuba/pull/1537 as a second major changeset and bundle them in a release, so testing in real repos is likely to be valuable as a combined exercise